components/centre_power.py

```diff
@@ -30,13 +30,19 @@ def draw_base(self, canvas: Canvas):
         pass
 
     def draw_data(self, canvas: Canvas, data: Data):
-        power = data["power"]
-        rec_power = data["rec_power"]
+        power = data["power"].get()
+        rec_power = data["rec_power"].get()
```

Comment:
Whoops, I thought we'd be able to get rid of these lines, didn't realise we still need the numerical values below. Ah well